### DISCUSSION TITLE
账号锁了，怎么解禁

### DISCUSSION THREAD

**Discussion by wuzhi-dev** [flagify-com/OctoMation#61] [General]
账号锁了，怎么解禁
用户多次尝试错误密码，导致账号被锁，怎么办？

**Comment by wzfukui** [comment]
耐心等待5分钟（默认），自动解禁，浏览器上有通知解禁时间。
这个时间可以通过系统运行环境变量调整，建议保持默认。

<img width="566" alt="image" src="https://github.com/flagify-com/OctoMation/assets/146505187/be5c3945-2096-4909-b312-da5afacb33e1">
